disnake/guild.py

```diff
@@ -3563,12 +3563,33 @@ async def kick(self, user: Snowflake, *, reason: Optional[str] = None) -> None:
         """
         await self._state.http.kick(user.id, self.id, reason=reason)
 
+    @overload
+    async def ban(
+        self,
+        user: Snowflake,
+        *,
+        clean_history_duration: Union[int, datetime.timedelta] = 86400,
```

Comment:
can this default to a timedelta instead?
```suggestion
        clean_history_duration: Union[int, datetime.timedelta] = datetime.timedelta(days=1),
```

Reply:
flake8 wouldn't like it due to https://flake8.codes/B008/, and while we could certainly make that work, I think an int default is fine